content/bn/docs/tasks/tools/included/kubectl-whats-next.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "এরপর কি?"
+description: "kubectl ইন্সটল করার পর কি হবে."
+headless: true
+_build:
+  list: never
+  render: never
+  publishResources: false
+---
+
+* [ইনস্টল করুন Minikube](https://minikube.sigs.k8s.io/docs/start/)
+* ক্লাস্টার তৈরি সম্পর্কে আরও জানতে দেখুন. [গাইড শুরু করা](/docs/setup/)
+* [আপনার অ্যাপ্লিকেশানটি কীভাবে লঞ্চ করবেন এবং প্রকাশ করবেন তা জানুন.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
+* আপনার যদি এমন একটি ক্লাস্টারে অ্যাক্সেসের প্রয়োজন হয় যা আপনি তৈরি করেননি, দেখুন
+  [Sharing Cluster Access document](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+* এটা পড়ুন [kubectl রেফারেন্স ডক্স](/docs/reference/kubectl/kubectl/)
```

Comment:
```suggestion
* [kubectl রেফারেন্স ডক্স](/bn/docs/reference/kubectl/kubectl/) পড়ুন ।
```

Reply:
Does that target page exist in the [dev-1.23-bn.1](https://github.com/kubernetes/website/tree/dev-1.23-bn.1) branch?

Reply:
Not yet @sftim , thus I have removed the bn folder